src/main/kotlin/org/cascadebot/cascadebot/events/GuildEvents.kt

```diff
@@ -56,12 +56,12 @@ class GuildEvents : ListenerAdapter() {
             } ?: run { greetings.welcomeChannel = null }
         }
 
-        if (greetings.welcomeDMEnabled) {
+        /*if (greetings.welcomeDMEnabled) {
             greetings.getRandomWelcomeDMMsg(event)?.let { message ->
                 event.user.takeUnless { it.isFake }
                         ?.let { channel -> channel.openPrivateChannel().queue(Consumer { it.sendMessage(message).queue() }, DiscordUtils.handleExpectedErrors(ErrorResponse.CANNOT_SEND_TO_USER)) }
             }
-        }
+        }*/
```

Comment:
Why has this been commented out?

Reply:
isFake is no longer a method, and I  don't know the replacement, I was going to ask you about this, but clearly forgot